scripts/tests/scripts_regression_tests.py

```diff
@@ -842,7 +842,7 @@ def test_ka_createnewcase_extra_machines_dir(self):
         cls._do_teardown.append(testdir)
 
         # Run case.setup
-        run_cmd_assert_result(self, "./case.setup",
+        run_cmd_assert_result(self, "./case.setup ",
```

Comment:
This extra space should be removed.

Reply:
Thanks @jedwards4b, just remove the extra space.